spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
>Suggest replacements please?
http://www.codeshare.io/XSJo1
In this case since all 3 describe have same precondition, can factored out and don't need a extra describe.

When we have tests which don't depend on this precondition, I think we should create a new file for them. ( and in this case we will probably not have those tests right ? )